demosplan/DemosPlanCoreBundle/Repository/ParagraphVersionRepository.php

```diff
@@ -254,10 +254,10 @@ public function generateObjectValues($entity, array $data)
             $entity->setText($data['text']);
         }
         if (array_key_exists('pId', $data)) {
-            $entity->setPId($data['pId']);
+            $entity->setProcedure($data['pId']);
         }
         if (array_key_exists('elementId', $data)) {
-            $entity->setElementId($data['elementId']);
+            $entity->setElement($data['elementId']);
```

Comment:
In both cases: same as above.
To solve this, it would be necessary to find all usages and search for actual incmoing type(s).

Reply:
done in [685364f](https://github.com/demos-europe/demosplan-core/pull/2960/commits/685364f97784d85dd520137c6bcee9f67c93e07c)

Reply:
You missed  `$entity->setProcedure($data['pId']);`
But anyway because this is the same case as
https://github.com/demos-europe/demosplan-core/pull/2960#discussion_r1569192780

Reply:
done in [f8f6053](https://github.com/demos-europe/demosplan-core/pull/2960/commits/f8f60532b56a758a0da95c3d345b88314f48427f)